etcd/etcd-env.sh

```diff
@@ -6,11 +6,11 @@ if test x$NAME == x; then
 fi
 export ETCD_NAME=${ETCD_NAME:-$HOSTNAME}
 export ETCD_DATA_DIR=/var/lib/etcd/${NAME}.etcd
```

Comment:
we already have `STATE_DIRECTORY` for that.  It gets a default value from `atomic` which is `/var/lib` when installing a system container and `%HOME/.data` when using `--user`.  Even if it gets a value from atomic, ${STATE_DIRECTORY} can be overriden at installation time, so you can do:

```bash
atomic install --system --set STATE_DIRECTORY=/mnt/somewhere/else etcd
```

and the etcd container will use the storage from `/mnt/somewhere/else/${NAME}.etcd`

It will be nice to allow to set the complete storage directory (without the ${NAME}.etcd part) but for allowing it and at the same time don't lose the capability of running multiple etcd instances, I think we will need to modify atomic to support variables expansion in the `defaultValues` block as well.